snapcraft/plugins/python.py

```diff
@@ -443,9 +443,14 @@ def list(self, exec_func=None):
         output = exec_func(cmd, env=self._env)
         package_listing = {}
         version_regex = re.compile('\((.+)\)')
+        version_regex_alt = re.compile('((\d+)\.)?((\d+)\.)?(\*|\d+)')
```

Comment:
`alt` is not a very good name. Maybe `version_regex_with_columns` ?
Also, I had no idea this columns thing existed. What about adding a few lines with a comment explaining why is this here?